var/spack/repos/builtin/packages/verilator/package.py

```diff
@@ -84,6 +85,14 @@ class Verilator(AutotoolsPackage):
     depends_on("perl", type=("build", "run"))
     depends_on("ccache", type=("build", "run"), when="@5.018:")
 
+    variant("systemc", default=False, description="Enable support for systemc")
+    variant("mold", default=False, description="Enable support for mold linker", when="@5.012:")
+    variant("ccache", default=False, description="Enable support for ccache builds", when="@5.018:")
+
+    depends_on("systemc", type=("build", "run"), when="+systemc")
+    depends_on("mold", type=("build", "run"), when="+mold")
+    depends_on("ccache", type=("build", "run"), when="+ccache")
```

Comment:
ccache is already added a few lines earlier.

Notice that both mold and ccache shouldn't really be dependencies, but I guess it's hard to disable them.

Reply:
> ccache is already added a few lines earlier.

Argh, sorry… my rebasing onto origin/develop lost the line remove… force-pushing.